packages/expo/src/modules/register/registerFirstStep/Register.tsx

```diff
@@ -0,0 +1,236 @@
+import React, {useState} from 'react';
+import Button from '@animavita/ui/core/Button/Button';
+import {Background} from '@animavita/ui/layout';
+import {useNavigation} from '@react-navigation/native';
+import {default as Slider} from '@animavita/ui/core/Form/Slider';
+import Input from '@animavita/ui/core/Form/Input';
+import Label from '@animavita/ui/core/Form/Label';
+import DropDownPicker from 'react-native-dropdown-picker';
+import {Formik} from 'formik';
+import * as Yup from 'yup';
+
+import Header from '../Header';
+import Error from '../Error';
+
+import {
+  Wrapper,
+  ScrollViewContainer,
+  LabelView,
+  LabelText,
+  ButtonsView,
+  PortButtonsView,
+  TypeButton,
+  TypeButtons,
+  TypeText,
+  PortButtons,
+  PortButton,
+  PortText,
+} from './styles';
+import {registerForm} from './registerForm';
+
+const validationSchema = Yup.object().shape({
+  name: Yup.string().required('O campo nome é obrigatório!'),
+  race: Yup.string().required('O campo raça é obrigatório!'),
+  observations: Yup.string().required('O campo observações é obrigatório!'),
+  gender: Yup.string().required('É obrigatório escolher o sexo do animal!'),
+  animalsSize: Yup.string().required('É obrigatório escolher um tipo de animal!'),
+  animalsType: Yup.string().required('Escolha o tamanho do animal!'),
+});
+
+const LARGE = 'large';
+
+interface Values {
```

Comment:
We already have all the types related to adoptions declared in the graphql package. So I think you can consume from there. Maybe we can extract all of common types in a new package and then consume from them. Does this make sense @renanmav?